app/(main)/graduation-progress/[studentCode]/[department]/page.tsx

```
@@ -229,6 +229,7 @@
             currentCredits={area.earned_credits}
             requiredCredits={area.required_credits}
             isCompleted={area.earned_credits >= area.required_credits}
+            requiredElectiveCredits={area.required_elective_courses}
             courses={area.courses}
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)